standup/style.css

```diff
@@ -132,12 +132,12 @@ body {
 .dates {
   position: sticky;
   top: 0;
-  padding: 1rem;
+  padding: 0.75rem;
```

Comment:
can't be do this in px, what is standard we follow here

Reply:
Previously, `rem` units were used over 'px' for their scalability and accessibility benefits. 'rem' and 'em' units are preferred due to their adaptability to various devices and screen sizes